Code/Editor/TrackView/AtomOutputFrameCapture.h

```diff
@@ -48,6 +50,8 @@ namespace TrackView
         AZStd::vector<AZStd::string> m_passHierarchy; //!< Pass hierarchy (includes pipelineName and CopyToSwapChain).
         CaptureFinishedCallback m_captureFinishedCallback; //!< Stored callback called from OnCaptureFinished.
 
+        bool m_pipelineCreated = false;
```

Comment:
In a change below you init with {} your bools, might be wise to unify styling, use {} or be explicit (I tend to prefer explicit init)